Bootstrap.php

```diff
@@ -381,6 +381,27 @@ public function update($oldVersion)
                 // Nothing to do
             case '4.0.0':
                 // Next release
+                $entityManager = $this->get('models');
+                $stripePaymentCardMethod = $entityManager->getRepository('Shopware\\Models\\Payment\\Payment')->findOneBy([
```

Comment:
Please import `Shopware\Models\Payment\Payment` as `PaymentMethod` and use it for loading the repository.

```suggestion
                $stripePaymentCardMethod = $entityManager->getRepository(PaymentMethod::class)->findOneBy([
```